docs/api/drivers/BusOut.md

```diff
@@ -2,7 +2,7 @@
 
 <span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_bus_out.png)<span>BusOut class hierarchy</span></span>
 
-Use the BusOut interface to create a number of [DigitalOut](digitalout.html) pins that you can write as one value. This API is useful for writing multiple outputs at once. You can use this API to write clearer code faster.
+Use the BusOut interface to combine a number of [DigitalOut](digitalout.html) pins to read them at once. This API is useful for writing to multiple pins together as single interface instead of individual pins.
```

Comment:
The first sentence says "to read them at once", although the BusOut API provides a read API its meant for reading whats being written.

Reply:
Updated it to "write" at once